Android/ARCore/source/Transforms/Metadata.xml

```diff
@@ -4,13 +4,15 @@
 	<attr path="/api/package[@name='com.google.ar.core.annotations']" name="managedName">Google.AR.Core.Annotations</attr>
 	<attr path="/api/package[@name='com.google.ar.core.exceptions']" name="managedName">Google.AR.Core.Exceptions</attr>
 
-	<attr path="/api/package[@name='com.google.vr.dynamite.client']" name="managedName">Google.VR.Dynamite.Client</attr>
```

Comment:
I see this is removed. Is this namespace no more? If so, then is there a reason for setting the base for `ObjectWrapper` below?

Reply:
typo probably